cypress/integration/pages/topicPage/index.js

```diff
@@ -17,9 +18,12 @@ Object.keys(config)
       describe(`${pageType} - ${currentPath}`, () => {
         before(() => {
           Cypress.env('currentPath', currentPath);
-
-          const newPath = `${currentPath}${overrideRendererOnTest()}`;
-
+          let newPath;
```

Comment:
would it be clearer like this? (the answer might be "no, not at all")

```
const newPath = `${currentPath}?id=${uuid() + Cypress.env('APP_ENV') !== 'live' ? '&renderer_env=live' : ''}`;
```

or 

```
const overrideSuffix = overrideRendererOnTest();
const newPath = `${currentPath + overrideSuffix + (overrideSuffix ? '&' : '?')}id=${uuid()}`;
```

Reply:
I like this. I have implemented it on another branch without the extra files.